### ISSUE TITLE
AttributeError: 'Edit' object has no attribute 'get_value' crash

### ISSUE THREAD

**Issue by fuho** [mitmproxy/mitmproxy#2776]
##### Steps to reproduce the problem:

1.Run mitmproxy, capture some flows containing json
2.Tab to go to `Response` to see the JSON
3. Hit `/` to Search and type in something, hit Enter

##### Expected behavior
Highlight first found search term

##### Current behavior
Mitmproxy crashes with Traceback (most recent call last):
```
  File "mitmproxy/tools/console/master.py", line 216, in run
  File "site-packages/urwid/main_loop.py", line 278, in run
  File "site-packages/urwid/main_loop.py", line 376, in _run
  File "site-packages/urwid/main_loop.py", line 682, in run
  File "site-packages/urwid/main_loop.py", line 719, in _loop
  File "site-packages/urwid/raw_display.py", line 393, in <lambda>
  File "site-packages/urwid/raw_display.py", line 493, in parse_input
  File "site-packages/urwid/main_loop.py", line 403, in _update
  File "site-packages/urwid/main_loop.py", line 503, in process_input
  File "mitmproxy/tools/console/window.py", line 308, in keypress
  File "site-packages/urwid/container.py", line 1116, in keypress
  File "mitmproxy/tools/console/statusbar.py", line 149, in keypress
  File "mitmproxy/tools/console/statusbar.py", line 104, in keypress
AttributeError: 'Edit' object has no attribute 'get_value'
```

#####

##### System information
```
Mitmproxy: 3.0.0.dev1136 (commit 15f525e) binary
Python:    3.5.3
OpenSSL:   OpenSSL 1.1.0g  2 Nov 2017
Platform:  Darwin-16.7.0-x86_64-i386-64bit
```

**Comment by mhils** [comment]
Thanks for the report!

**Comment by MatthewShao** [comment]
I try to reproduce this bug and found that it raise the same exception on every flow, no matter whether it contains json or not.

**Comment by fuho** [comment]
That is true, I wrote it for the one I actually tested just in case it was
something specific.

**Comment by ujjwal96** [comment]
```
  File "/home/ujjwal/Github/mitmproxy/mitmproxy/tools/console/window.py", line 308, in keypress
    k = super().keypress(size, k)
  File "/home/ujjwal/Github/mitmproxy/venv/lib/python3.5/site-packages/urwid/container.py", line 1116, in keypress
    return self.footer.keypress((maxcol,),key)
  File "/home/ujjwal/Github/mitmproxy/mitmproxy/tools/console/statusbar.py", line 150, in keypress
    return self.ab.keypress(*args, **kwargs)
  File "/home/ujjwal/Github/mitmproxy/mitmproxy/tools/console/statusbar.py", line 105, in keypress
    self.prompt_execute(self._w.get_text())
AttributeError: 'CommandEdit' object has no attribute 'get_text'
```

I get this error when trying commands invoked by <kbd>W</kbd> and <kbd>L</kbd>
I think this issue applies to most of the commands that take a text input

**Comment by mhils** [comment]
This has just been fixed by @ujjwal96 in #2780! :)

**Comment by fuho** [comment]
Wow, super quick response and fix. Thanks a lot!
